ResearchKit.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+  s.name         = 'ResearchKit'
+  s.version      = '1.0.0'
+  s.summary      = 'ResearchKit is an open source software framework that makes it easy to create apps for medical research or for other research projects.'
+  s.homepage     = 'https://www.github.com/ResearchKit/ResearchKit'
+  s.documentation_url = 'http://researchkit.github.io/docs/'
+  s.license      = { :type => 'BSD', :file => 'LICENSE' }
+  s.author       = { "Apple Inc." => "http://apple.com" }
+  s.source       = { :git => 'https://github.com/ResearchKit/ResearchKit.git', :tag => "v#{s.version}"}
+  s.public_header_files = `./scripts/find_headers.rb --public`.split("\n")
+  s.private_header_files = `./scripts/find_headers.rb --private`.split("\n")
```

Comment:
I've tested this and it doesn't do what I expected. In Xcode, frameworks can have public headers, private headers, and project headers. Here, private headers seems to be an exclusion glob against public headers, which excludes some headers from the public header directory.

I think we need to remove the private_header_files line. Sorry I mis-directed you earlier - the naming led me to expect it would behave like Xcode.